LaTeX/LaTeX.sublime-syntax

```diff
@@ -198,6 +207,123 @@ contexts:
     - include: paragraph-pop
     - include: else-pop
 
+  xparse-argspec:
+    - match: (?=\{)
+      set:
+        # deliberately using an anonymous scope here. we need two match the `{`
+        # in two stages, so we can apply the `clear_scopes`, and still be able
+        # to push `xparse-argspec` onto the stack without messing up scoping.
+        - clear_scopes: 1
+        - match: '\{'
+          scope: punctuation.definition.group.brace.begin.latex
+          set: xparse-argspec-content
+
+    - include: paragraph-pop
+    - include: else-pop
+
+  xparse-argspec-content:
+    - clear_scopes: 1
+    - meta_scope: meta.function.parameters.latex
+    - include: brace-group-end
+    - include: macro-braces
+    - include: xparse-single-argspec
+    # not actually allowed, but included here to future-proof highlighting
+    - include: general-constants
+    - include: general-commands
+    - include: paragraph-pop
+
+  xparse-single-argspec:
+    # modifiers and simple specs
+    - match: '[+!movsb]'
+      scope: string.unquoted.latex meta.annotation.parameters.latex
+
+    - match: t
```

Comment:
Do we probably need to check for word boundaries to prevent matching letters within unmatched tokens?

```suggestion
    - match: t\b
```

Reply:
This is a valid declaration of a new command:
```
\NewDocumentCommand{\MyMacro}{tm}{\IfBooleanTF{#1}{A}{B}}
```
This will result in `A` is used as `\MyMacro m` and in `B` is used as `\MyMacro`, i.e. the letter `m` is the token that the t-type argument will look for here. So `t` should be scoped as an argument specifier, whereas the `m` is the literal thing that is looked for.

Reply:
Ok, so it basically follows the way how shellscript can chain flags. Thank's for clarification.